service/src/ocall_bridge/sidechain_ocall.rs

```diff
@@ -0,0 +1,81 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+	Copyright (C) 2017-2019 Baidu, Inc. All Rights Reserved.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::{
+	ocall_bridge::bridge_api::{OCallBridgeError, OCallBridgeResult, SideChainBridge},
+	sidechain_storage::BlockStorage,
+	sync_block_gossiper::GossipBlocks,
+};
+use codec::Decode;
+use its_primitives::types::SignedBlock as SignedSidechainBlock;
+use log::*;
+use std::sync::Arc;
+
+pub struct SideChainOCall<S, D> {
```

Comment:
Now I wonder. Did you make it on purpose with the capital 'C's in the CamelCase names? I'd prefer it to be consistent with the snake_case versoin.